src/cow-react/modules/limitOrders/hooks/useGetInitialPrice.ts

```diff
@@ -9,8 +9,14 @@ import { parsePrice } from '@cow/modules/limitOrders/utils/parsePrice'
 import useIsWindowVisible from 'hooks/useIsWindowVisible'
 import { useWalletInfo } from '@cow/modules/wallet'
 import { getNativePrice } from '@cow/api/gnosisProtocol'
+import { ApiError } from '@cowprotocol/cow-sdk'
 
-type PriceResult = number | Error | undefined
+type ErrorWithApiError = { errorType: string; description: string; rawApiError: ApiError }
+type PriceResult = number | Error | undefined | ErrorWithApiError
+
+function hasApiError(value: any): value is ErrorWithApiError {
+  return value && typeof value === 'object' && value.errorType && value.description && value.rawApiError
```

Comment:
Looks good for now. The change won't be needed after this:
https://github.com/cowprotocol/cow-sdk/pull/122/files#diff-7633654c6f288b7d31db55cbe6c975509378f0b8830b6fc0ffb694e3efc38b76R4